make_index.py

```diff
@@ -83,10 +84,10 @@ def main(argv: Sequence[str] | None = None) -> int:
         for filename in filenames
     ):
         basename = os.path.basename(filename)
-        if basename in on_pypi:
-            raise AssertionError(f"{basename}: already on pypi?")
-        elif basename in seen:
-            continue
+        #if basename in on_pypi:
+        #    raise AssertionError(f"{basename}: already on pypi?")
+        #elif basename in seen:
+        #    continue
```

Comment:
this is a really important safeguard and should not be commented out

Reply:
oh this is a really old branch, iirc i was doing some hacky things to generate a local index but now im just using `tests/make_index_test.py`